src/dispatch/alembic/versions/62465f508f69_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 62465f508f69
+Revises: 1221a4d60f03
+Create Date: 2020-04-21 15:24:55.512154
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '62465f508f69'
+down_revision = '1221a4d60f03'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('dispatch_user',
+    sa.Column('email', sa.String(), nullable=False),
+    sa.Column('password', sa.Binary(), nullable=True),
```

Comment:
Should we allow people to create accounts without password?

Reply:
Probably not.